vic/vic_run/src/surface_fluxes.c

```diff
@@ -261,6 +261,7 @@ surface_fluxes(bool                 overstory,
     coverage = snow->coverage;
     snow_energy = (*energy);
     soil_energy = (*energy);
+    iter_soil_energy = (*energy);
```

Comment:
I think it is OK - but the logic is hard to read in that loop and would benefit from a refactor (but not as part of this PR). For example 
```C 
define firstloop 999
```
and then replace `999` by `firstloop` may make this easier to read, but not here. I'm fine with the PR as is, so go ahead and merge (or I can do so).